cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -21,3 +21,34 @@ def test_embedded_ansible_enable(enabled_embedded_appliance):
     assert wait_for(func=lambda: enabled_embedded_appliance.is_nginx_running, num_sec=30)
     assert enabled_embedded_appliance.ssh_client.run_command(
         'curl -kL https://localhost/ansibleapi | grep "Ansible Tower REST API"')
+
+
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.uncollectif(lambda: current_version() < "5.8")
+def test_embedded_ansible_disable(enabled_embedded_appliance):
+    """Tests wether the embedded ansible role and all workers have stopped correctly"""
+    assert wait_for(func=lambda: enabled_embedded_appliance.is_rabbitmq_running, num_sec=30)
+    assert wait_for(func=lambda: enabled_embedded_appliance.is_nginx_running, num_sec=30)
+    enabled_embedded_appliance.disable_embedded_ansible_role()
+
+    def is_superviserd_stopped(enabled_embedded_appliance):
```

Comment:
**OPTIONAL:** I think the arg to the function unneeded as it should use the appliance from the outer scope.